src/auth/auth.service.ts

```diff
@@ -301,12 +305,37 @@ export class AuthService {
   }
 
   deleteUser = async (email: string) => {
-    const resp = await this.userRepo.delete({ email });
-    if (resp.affected == 0) {
+    try {
+      const user = await this.userRepo.findOne({
+        where: { email },
+      });
+
+      if (!user) {
+        throw new BadRequestException(
+          ZaLaResponse.BadRequest(
+            'User Not found',
+            'The Email Does not exists',
+            '400',
+          ),
+        );
+      }
+      //  send Delete request to the profile service to delete the user profile
+      // Axios
+      const coreUrl = `${this.configService.get<string>(
+        configConstant.baseUrls.coreService,
+      )}/profile/${user.profileID}`;
+      console.log(coreUrl);
+
+      await this.httpService.axiosRef({
+        method: 'delete',
```

Comment:
And you need to confirm the response from this call before you delete the user

Reply:
@PhilemonBrain  The user should be deleted before the call to delete the profile or the reverse?
Which of them should come first?

Because i just updated the file now to delete the user before making the call based on chimex suggestion